### ISSUE TITLE
Don't force port 22 for rsync backup

### ISSUE THREAD

**Issue by SoMuchForSubtlety** [mail-in-a-box/mailinabox#2104]
Currently, the backup script has port 22 hardcoded.
https://github.com/mail-in-a-box/mailinabox/blob/3a7de051ee1e87d08589a946e3d05bbf6f6806fc/management/backup.py#L19

This prevents setting a non-standard port via `~/.ssh/config` and I don't see any upsides since port 22 is the default anyway.

I need to use port 23 since that is what's being used by [hetzner storage boc](https://docs.hetzner.com/robot/storage-box/access/access-ssh-rsync-borg).

**Comment by JoshData** [comment]
Yeah I can't think of a reason why `-p 22` should be there, and I would accept a pull request removing it.

**Comment by schlypel** [comment]
@SoMuchForSubtlety this is merged in 61 and works now, can this be closed?
Thanks.
